apps/dav/lib/Search/EventsSearchProvider.php

```diff
@@ -157,8 +157,13 @@ public function search(
 				$calendar = $subscriptionsById[$eventRow['calendarid']];
 			}
 			$resourceUrl = $this->getDeepLinkToCalendarApp($calendar['principaluri'], $calendar['uri'], $eventRow['uri']);
+			$result = new SearchResultEntry('', $title, $subline, $resourceUrl, 'icon-calendar-dark', false);
 
-			return new SearchResultEntry('', $title, $subline, $resourceUrl, 'icon-calendar-dark', false);
+			$dtStart = $component->DTSTART;
+			$startDateTime = $dtStart->getDateTime()->format('U');
+			$result->addAttribute("createdAt", $startDateTime);
```

Comment:
We can probably fix the notices like this:
```suggestion
			if ($dtStart instanceof Date) {
				$startDateTime = $dtStart->getDateTime()->format('U');
				$result->addAttribute("createdAt", $startDateTime);
			}
```

Reply:
Which date is this?
> use Sabre\VObject\Property\ICalendar\Date;

did not work